src/components/post-form/post-form.tsx

```diff
@@ -1,266 +1,97 @@
-import { CalendarIcon } from "lucide-react";
 import Link from "next/link";
-import { FC, useContext } from "react";
-import type { SelectSingleEventHandler } from "react-day-picker";
-import { UseFormRegister, useForm } from "react-hook-form";
+import { FC } from "react";
 import { BiArrowBack } from "react-icons/bi";
-import { MdAdd, MdRemove } from "react-icons/md";
 
-import { PosterContext } from "@/context/poster";
-import { cn } from "@/lib/utils";
-import { formatFullDate } from "@/utils/utils";
-
-import { Button } from "../ui/button";
-import { Calendar } from "../ui/calendar";
-import { Input } from "../ui/input";
-import { Label } from "../ui/label";
-import { Popover, PopoverContent, PopoverTrigger } from "../ui/popover";
 import {
-  Select,
-  SelectContent,
-  SelectItem,
-  SelectTrigger,
-  SelectValue,
-} from "../ui/select";
```

Comment:
So everything was changed for accordions

Reply:
Accordions are containers. They consist of a title container which is always visible, as well as some kind of indicator that the container is clickable and is not expanded. When expanded, the hidden container shows itself. You can see an example of it [here](https://ui.shadcn.com/docs/components/accordion).